src/test/java/org/springframework/integration/kafka/config/xml/KafkaInboundChannelAdapterParserTests-context.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:int="http://www.springframework.org/schema/integration"
+	xmlns:int-kafka="http://www.springframework.org/schema/integration/kafka"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+		http://www.springframework.org/schema/integration http://www.springframework.org/schema/integration/spring-integration.xsd
+		http://www.springframework.org/schema/integration/kafka http://www.springframework.org/schema/integration/kafka/spring-integration-kafka.xsd">
+
+	<int-kafka:inbound-channel-adapter
+			id="adapter1"
+			consumer-factory="consumerFactory"
+			ack-factory="ackFactory"
+			topics="topic1"
+			channel="inbound"
+			client-id="client"
+			group-id="group"
+			message-converter="converter"
+			payload-type="#{T(java.lang.String)}"
```

Comment:
Gary, I wonder if we should reconsider that `Type payloadType` in the `KafkaMessageSource` into the `Class<?>`.

Different issue though...

Merging this one with some minor polishing!